index.html

```diff
@@ -27,15 +26,249 @@ <h1 class="intro">I'm Morteza Glad to See You!</h1>
+                        <div class="text">2015</div>
+                    </div>
+                </div>
+    
+                <div class="desc">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+    
+                <div class="tags">
+                    <div>
+                        <li class="tag"><a href="">html</a></li>
+                    </div>
+                    <div>
+                        <li class="tag"><a href="">CSS</a></li>
+                    </div>
+                    <div>
+                        <li class="tag"><a href="">Javascript</a></li>
+                    </div>
+                </div>
+    
+                <div class="action">
+                    <div class="btn-action">
+                        <a class="see">See Project</a>
+                    </div>
+                </div>
+            </div>
+    
+    
+        </section>
+        <section class="works">
+            <div class="snapshot-4"></div>
+         
+            <div class="info">
+               <div class="title">Tonic</div> 
+                <div class="frame">
+                    <div class="client">
+                        <div class="name">
+                            CANOPY
+                        </div>
+                    </div>
+                    <div class="counter"></div>
+    
+                    <div class="role">
+                        <div class="text">Back End Dev</div>
+                    </div>
+                    <div class="counter"></div>
+    
+                    <div class="role">
+                        <div class="text">2015</div>
+                    </div>
+                </div>
+    
+                <div class="desc">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+    
+                <div class="tags">
+                    <div>
+                        <li class="tag"><a href="">html</a></li>
+                    </div>
+                    <div>
+                        <li class="tag"><a href="">CSS</a></li>
+                    </div>
+                    <div>
+                        <li class="tag"><a href="">Javascript</a></li>
+                    </div>
+                </div>
+    
+                <div class="action">
+                    <div class="btn-action">
+                        <a class="see">See Project</a>
+                    </div>
+                </div>
+            </div>
+    
+    
+        </section>
+
+
+        <section class="about">
+            <div class="myself">About Myself</div>
+            <p class="content"> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <div class="connect">LET'S CONNECT</div>
```

Comment:
- **[OPTIONAL]** This is a link to your contact form, please 🙏🏽 add the `a` tag.